src/main/java/org/jabref/gui/EntryTypeViewModel.java

```diff
@@ -180,7 +182,13 @@ public void runFetcherWorker() {
             } else if (result.isEmpty()) {
                 String fetcher = selectedItemProperty().getValue().getName();
                 String searchId = idText.getValue();
-                dialogService.showErrorDialogAndWait(Localization.lang("Fetcher '%0' did not find an entry for id '%1'.", fetcher, searchId));
+                boolean addEntryFlag = dialogService.showConfirmationDialogAndWait("DOI not found",
+                        Localization.lang("Fetcher '%0' did not find an entry for id '%1'.", fetcher, searchId),
+                        "Add entry manually", "Return to dialog");
```

Comment:
Please also Add the localization.lang here for all options. https://devdocs.jabref.org/getting-into-the-code/code-howtos#using-localization-correctly

Reply:
@Siedlerchr , I added the language keys to JabRef_en.properties